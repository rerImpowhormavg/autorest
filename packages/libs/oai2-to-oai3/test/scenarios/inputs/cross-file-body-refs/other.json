{
  "parameters": {
    "BodyParam": {
      "in": "body",
      "name": "MyBodyParam",
      "required": true,
      "schema": {
        "$ref": "#/definitions/MyBodySchema"
      }
    }
  },
  "definitions": {
    "MyBodySchema": {
      "type": "object",
      "properties": {
        "url": {
          "type": "string"
        }
      }
    }
  }
}
