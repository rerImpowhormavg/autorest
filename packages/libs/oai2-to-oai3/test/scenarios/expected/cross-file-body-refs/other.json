{
  "servers": [],
  "components": {
    "schemas": {
      "MyBodySchema": {
        "type": "object",
        "properties": {
          "url": {
            "type": "string"
          }
        }
      }
    }
  }
}